app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -37,6 +37,37 @@ const ExperimentForm: React.FC = () => {
   );
   const history = useHistory();
 
+  const featureOptions = [
+    { value: "FEATURE 1", label: "FEATURE 1" },
+    { value: "FEATURE 2", label: "FEATURE 2" },
+  ];
+
+  const audienceOptions = [
+    { value: "AUDIENCE 1", label: "AUDIENCE 1" },
+    { value: "AUDIENCE 2", label: "AUDIENCE 2" },
+  ];
+
+  const handlefeatureChange = (featureValues) => {
+    const formattedFeatures = featureValues.map((feature) => feature.value);
+    dispatch({
+      type: ExperimentReducerActionType.UPDATE_STATE,
+      state: {
+        ...formData,
+        features: formattedFeatures,
+      },
+    });
+  };
+
+  const handleAudienceChange = (audienceValue) => {
+    dispatch({
+      type: ExperimentReducerActionType.UPDATE_STATE,
+      state: {
+        ...formData,
+        audience: audienceValue.value,
+      },
+    });
+  };
```

Comment:
You can't reuse the `handleChange` below for this?

Reply:
we could if we decide to save feature and audience values as 
`{"value": "feature value", "label": "feature label" }`?
Do we prefer that?

Reply:
Hrmmm yeah we have to store the whole object somewhere to display the right values, but we only want to actually store the value in the data object that gets posted back to the server.  Maybe there's some way we can hide that logic inside the implementation of the Select component?